src/core/inputs/amqp-input.ts

```diff
@@ -165,7 +165,7 @@ export default class CoreInputAmqp implements IInputProvider {
       if (this.options.ackMode === 'auto') {
         channel.ack(msg);
       }
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
With the typescript update these catches were causing build issues as unkown type is not allowed.